include/boost/geometry/core/config.hpp

```diff
@@ -14,6 +14,16 @@
 
 #include <boost/config.hpp>
 
+// Flip around the default. This is targeted to tests. As soon as the default changes,
+// some unit tests (tagged by _alternative) are tested with the non-default.
+#if defined(BOOST_GEOMETRY_ROBUSTNESS_ALTERNATIVE)
+  #if defined(BOOST_GEOMETRY_NO_ROBUSTNESS)
+    #undef BOOST_GEOMETRY_NO_ROBUSTNESS
+  #else
+    #define BOOST_GEOMETRY_NO_ROBUSTNESS
+  #endif
+#endif
```

Comment:
This is because I add more and more tests which are "ready", and get the _alternate state now.
As soon as we switch, they test the other way round because for a while we'll have to support old behavior too.